Kernel/CMakeLists.txt

```
@@ -451,6 +430,9 @@ if (NOT "${SERENITY_ARCH}" STREQUAL "aarch64")
     add_library(kernel_heap STATIC ${KERNEL_HEAP_SOURCES})
 endif()
 add_executable(Kernel ${SOURCES})
+
+include("${CMAKE_SOURCE_DIR}/Kernel/Modules.cmake")
```

Probably best to not hard-code the project structure in the CMake lists. I did put some effort into making all of the `.cmake` files live in Meta/CMake though, so moving the file there and calling it `KernelModules.cmake` or some such would let us `include(KernelModules)`

```suggestion
include("${CMAKE_CURRENT_LIST_DIR}/Modules.cmake")
```

---

You're absolutely right about this. I will also look into how to move the file to the Meta folder, although it might a bit of work. We will see how it goes :)